release-team/role-handbooks/release-notes/README.md

```diff
@@ -183,16 +182,16 @@ Begin running release-notes tool for ongoing collection of release notes with th
 
 ### Week 11 (Begin of Code Freeze)
 
-- Create Google Doc for generated release notes and share with release-notes team
+- Create Google Doc for generated release notes and share with the release-notes team
```

Comment:
I don't think this is true anymore -- we've moved on from Google Doc to the Markdown in releases directory of this repository.

Reply:
I am removing it from the README